grammars/suggestions-en.yaml

```
@@ -0,0 +1,199 @@
+1:
+ - print
+ - ask
+ - echo
+ - turn
+ - forward
```

Would it make sense to write it like

```suggestion
1: [print, ask, echo, turn, forward]
```

? That makes it easier to compare the list of keywords between levels, which may reduce the risk that you forget a keyword.

---

I agree that is a bit more readable but I don't think it is a blocker at this point

---

Made a separate issue for this: #1350